Controller/RestController.php

```diff
@@ -14,16 +19,26 @@ class RestController extends AbstractController
 {
     use CsrfCheckerTrait;
 
+    public function __construct(
+        protected DataGridRequestHandler $dataGridRequestHandler,
```

Comment:
this will not work with php 7.4, as far as composer json require < v8

Reply:
The policy is to have a branch per symfony version

Reply:
@94noni @tuxes3  Master can be used for latest and greatest, but only once the branches for older versions have been made

Reply:
then this PR should update the composer require entry to have only PHP v8+ isn't it?

Reply:
Correct

Reply:
IMH we should not use this syntax for now to keep compatibility with php 7.4 and keep compatibility with Symfony 5 for now

Reply:
@Seb33300 that would be confusing as we now have the 5.2 branch for that

Reply:
I am not sure to understand where is the issue